services/web/server/src/simcore_service_webserver/login/_constants.py

```diff
@@ -49,6 +49,9 @@
 ] = "This account has expired and does not have anymore access. Please contact support for further details: {support_email}"
 MSG_WRONG_2FA_CODE: Final[str] = "Invalid code (wrong or expired)"
 MSG_WRONG_PASSWORD: Final[str] = "Wrong password"
+MSG_WEAK_PASSWORD: Final[
+    str
+] = "Password must be at least {LOGIN_PASSWORD_MIN_LENGTH} characters long"
```

Comment:
I think you are missing "f" before the string

Reply:
Actually I dont think so since it is a "Final" constant string. We use the `.format` call on it to add the content of `LOGIN_PASSWORD_MIN_LENGTH`. Maybe @pcrespov can comment?